README.md

```diff
@@ -14,11 +14,9 @@ See [Running with Docker](docs/running-with-docker.md)
 
 ### Prerequisites
 
-#### Postgres
+Use Homebrew to install Postgres and Redis on MacOS:
 
-Use Homebrew to install Postgres on MacOS:
-
-`$ brew install postgres`
+`$ brew install postgres redis`
 
 The application requires a user named `postgres` so create that user if it does not exist already:
```

Comment:
If you've installed with `brew install postgres` then the user `postgres` will exist as the only user and a `SUPERUSER`. If you use the `rake db:reset_to_production` task then you'll need a `root` user, same as RDS, but the instructions are [elsewhere](https://github.com/dxw/DataSubmissionServiceAPI/blob/develop/docs/get-prod-db-backup.md#restore-the-backup) for that :)